core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -51,7 +51,8 @@ class TestRaftServer(
 ) extends Logging {
   import kafka.tools.TestRaftServer._
 
-  private val partition = new TopicPartition("__cluster_metadata", 0)
+  private val partition = new TopicPartition("__raft_performance_test", 0)
+  private val topicId = new Uuid(0L, 2L)
```

Comment:
Could we add a comment here which explains that this value must be constant and that the value is chosen not to conflict with the topicId used for `@metadata`?